### DISCUSSION TITLE
Maintainers ALERT

### DISCUSSION THREAD

**Discussion by alexandermirzoyan** [react-grid-layout/react-resizable#243] [Q&A]
Just want to see if maintainers are active here. As there are a lot of unanswered opened issues and questions.

Please let us know how to fix our problems.

**Comment by STRML** [comment]
If you'd like to answer some of them or submit/review PRs, be my guest.

The vast majority of open PRs right now are irrelevant. Issues take more work. I don't have time for them right now.

OSS is free.

**Reply by alexandermirzoyan** [reply]
ahah yeah I know that OSS is free. 

And I understand that you may not have time on colaborating. This was a test message to get in touch with someone. Thank you for your response. But the problem that I have faced is about this issue, if you would find time please check it. 

https://github.com/react-grid-layout/react-resizable/discussions/221

**Reply by STRML** [reply]
It does not seem like you've spent time trying to solve it in any way. 

OSS requires mutual respect. If you want a feature, implement it.
